### PR TITLE
move from error to warn

### PR DESCRIPTION
**Pull request by kishansairam9**

### CODE REVIEW

File: helm/values.yaml [hunk 1]
```diff
@@ -508,7 +508,7 @@ server:
       type: pd-standard
 
   logging:
-    level: error
+    level: warn
```
Comment (ravisingal) [inline helm/values.yaml:511]: it already is set to `error` in helm-values repo.

Comment (kishansairam9) [reply helm/values.yaml:511]: We will adjust in helm-values repo as needed

File: helm/templates/controller/log-configmap.yaml [hunk 2]
```diff
@@ -11,7 +11,7 @@ metadata:
     heritage: {{ .Release.Service }}
 data:
   pinot-controller-log4j2.xml: |-
-    <Configuration>
+    <Configuration monitorInterval="30">
```
Comment (kishansairam9) [inline helm/templates/controller/log-configmap.yaml:14]: same with this, we will be modifying in helm-values

File: helm/values.yaml (hunk 1 above)
Comment (kishansairam9) [reply helm/values.yaml:511]: https://github.com/Traceableai/helm-values/pull/4139

Comment (ravisingal) [reply helm/values.yaml:511]: what I meant was that we already have set the logging level to `error`. are there any specific warn logs we are interested in? there were too much logging by pinot.

Comment (kishansairam9) [reply helm/values.yaml:511]: Laxman and mihir were of opinion we should have warn logs and fix any underlying issues

Comment (laxmanchekka) [reply helm/values.yaml:511]: I only suggested better to keep warn as the default.
If they are too noisy, we may need to investigate and fix or turn them off at package level in worst case.

File: helm/templates/broker/log-configmap.yaml [hunk 3]
```diff
@@ -25,7 +25,7 @@ data:
           <AppenderRef ref="console"/>
         </Root>
         <AsyncLogger name="org.reflections" level="error" additivity="false"/>
-        <AsyncLogger name="org.apache.pinot.broker.querylog.QueryLogger" level="info"/>
+        <Logger name="org.apache.pinot.broker.querylog.QueryLogger" level="info" additivity="false"/>
```
Comment (laxmanchekka) [inline helm/templates/broker/log-configmap.yaml:28]: Logger need to have console appenderRef.

**Review by laxmanchekka — APPROVED** [review]

**Pull request merged**